<launch>

    <group ns="agent_2">
    
        <rosparam command="load" file="$(find offset_etddf)/config/ros_config.yaml" />

        <node name="planner0" pkg="cohrint_minau" type="planner.py" output="screen" />
	    <node name="sensor_pub" pkg="cohrint_minau" type="publish_sensors.py" output="screen" />
	    <node name="comms_module" pkg="cohrint_minau" type="comms.py" />
	    <node name="agent" pkg="cohrint_minau" type="agent_wrapper.py" />
    
    </group>

</launch>